extensions/default/src/DataSourceConfigurationAPI/GoogleCloudDataSourceConfigurationAPI.ts

```diff
@@ -0,0 +1,192 @@
+import { ExtensionManager, Types } from '@ohif/core';
+
+/**
+ * The various Google Cloud Healthcare path item types.
```

Comment:
can we add a comment that you should look aththe Base.... to write your own custom one